README.md

```diff
@@ -1,4 +1,70 @@
-teleop_twist_joy [![Build Status](https://travis-ci.org/ros-teleop/teleop_twist_joy.svg?branch=indigo-devel)](https://travis-ci.org/ros-teleop/teleop_twist_joy)
+ros2/teleop_twist_joy [![Build Status](https://travis-ci.org/ros-teleop/teleop_twist_joy.svg?branch=indigo-devel)](https://travis-ci.org/ros-teleop/teleop_twist_joy)
```

Comment:
The link to the travis job here is wrong.  I think we should just remove the link completely, since we don't use travis on this repository at all.